projekt_do_pracy_na_labach/komendy_lab1.txt

```diff
@@ -0,0 +1,39 @@
+#Komendy
+
+### 3.0 - Hello world w Dockerfile 
+# docker build -t "nazwa1" ./
+
+### 3.5 - Otworzyć aplikację na świat. 
+# docker run -p 8001:8000 nazwa1
+
+### 4.0 - Połączyć aplikację back-endową z bazą danych
+# docker network create siec1
+# docker run -d --network siec1 -e DATABASE_URL="mongodb://baza_danych_1:27017" nazwa1
+# docker run -d --network siec1 --name baza_danych_1 mongo
+# docker network inspect siec1
+# docker run -d --network siec1 -e DATABASE_URL="mongodb://baza_danych_1:27017" nazwa1
+# docker run -d -p 8001:8000 --network siec1 -e DATABASE_URL="mongodb://baza_danych_1:27017" nazwa1
+
+### 4.5 - Dodanie wolumenów do Dockerfile'a z bazą danych
+# docker volume create volume_1
+# docker run -d --name backend_v2 -v volume_1:/app nazwa1 fe0281c6c321b6f5b01b68f784708cdf8541305759300114b2528508a3ed56df
```

Comment:
Wolumen w zadaniu na 4.5 miał się tyczyć obrazu z bazą danych, nie obrazu backendu. 
W sensie proces tworzenia jest ok ze strony technicznej, ale nie tego tyczyło się zadanie. 
Jak stworzysz wolumen dla bazy danych w miejscu gdzie ona trzyma dane to zaliczę :)